api/middleware/authStrategies.js

```diff
@@ -7,16 +7,51 @@ const TwitterStrategy = require('passport-twitter').Strategy;
 const models = require('../../database/models');
 const keys = require('../../config/secret');
 
-async function callbackStrategy(profile, cb) {
-  const userEmail = await models.Users.findOne({
-    where: { email: profile.email }
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((id, done) => {
+  models.Users.findByPk(id, (err, user) => {
+    console.log('deserialize', user);
```

Comment:
can you take out this console.log statement

Reply:
Fixed.